next.config.mjs

```diff
@@ -10,7 +10,7 @@ const nextConfig = {
         },
         {
           source: "/transcripts",
-          destination: "/gh-pages/index.html",
+          destination: "/categories",
```

Comment:
revert this change, and keep transcripts as is.
These are rewrite rules and they do not redirect, the pattern we'd want in this case would be a redirect.

Rather than a redirect, I want `/transcripts` to be an entry point to the old build. So leave it as is
However, in the landing page, footer, and header, update all links that point to `/transcripts` to point to `/categories` instead

Reply:
This wasn't resolved

Reply:
cc: @0tuedon